[EDU-2571] Add L2 region info (#290)
aziontech/docs: src/content/docs/pt-br/pages/produtos/edge-application/l2-caching/index.mdx

permalink: /documentacao/produtos/edge-application/l2-caching/
---
import Button from '~/components/Button.astro';
import Badge from '~/components/Badge.astro';

<Badge variant="accent">

> Early Access

Reviewer: ```suggestion
Preview
```

> <Badge variant="accent">
> Early Access
> </Badge>

Author: ```suggestion

```

## Regiões

Quando você ativa o **L2 Caching** como uma camada extra de cache, os servidores edge dessa segunda camada de cache são hospedados nos EUA (`na-united-states`) por padrão. 

> A Azion também oferece a opção de hospedar os servidores L2 no Brasil (`sa-brazil`). Para alternar entre as regiões em suas aplicações de borda, entre em contato com a [equipe de vendas](https://www.azion.com/en/contact-sales/).

Reviewer 2: ```suggestion
A Azion também oferece a opção de hospedar os servidores L2 no Brasil (`sa-brazil`). Para alternar entre as regiões em suas edge applications, entre em contato com a [equipe de vendas](https://www.azion.com/en/contact-sales/).
```
